sdk/core/src/prelude.rs

```diff
@@ -2,32 +2,33 @@ pub use crate::ba512_range::BA512Range;
+    BlockIdSupport, BodyRequired, BodySupport, CacheControlOption, CacheControlSupport,
+    ChunkSizeOption, ChunkSizeSupport, ClientRequestId, ClientRequestIdOption,
+    ClientRequestIdSupport, ContainerNameRequired, ContainerNameSupport, ContentDispositionOption,
+    ContentDispositionSupport, ContentEncodingOption, ContentEncodingSupport,
+    ContentLanguageOption, ContentLanguageSupport, ContentLengthOption, ContentLengthRequired,
+    ContentLengthSupport, ContentMD5Option, ContentMD5Support, ContentType, ContentTypeOption,
+    ContentTypeRequired, ContentTypeSupport, Continuation, ContinuationOption, ContinuationSupport,
+    DeleteSnapshotsMethod, DeleteSnapshotsMethodSupport, DelimiterOption, DelimiterSupport,
+    HttpClient, IfMatchConditionOption, IfMatchConditionSupport, IfModifiedSince,
+    IfModifiedSinceOption, IfModifiedSinceSupport, IfSinceConditionOption, IfSinceConditionSupport,
     IfSourceMatchConditionOption, IfSourceMatchConditionSupport, IfSourceSinceConditionOption,
     IfSourceSinceConditionSupport, IncludeCopyOption, IncludeCopySupport, IncludeDeletedOption,
     IncludeDeletedSupport, IncludeListOptions, IncludeMetadataOption, IncludeMetadataSupport,
     IncludeSnapshotsOption, IncludeSnapshotsSupport, IncludeUncommittedBlobsOption,
     IncludeUncommittedBlobsSupport, IsSynchronousOption, IsSynchronousSupport,
     LeaseBreakPeriodOption, LeaseBreakPeriodRequired, LeaseBreakPeriodSupport,
     LeaseDurationRequired, LeaseDurationSupport, LeaseIdOption, LeaseIdRequired, LeaseIdSupport,
-    MaxResultsOption, MaxResultsSupport, MetadataOption, MetadataSupport, NextMarkerOption,
-    NextMarkerSupport, PageBlobLengthRequired, PageBlobLengthSupport, PrefixOption, PrefixSupport,
-    ProposedLeaseIdOption, ProposedLeaseIdRequired, ProposedLeaseIdSupport, RangeOption,
-    RangeRequired, RangeSupport, SequenceNumberConditionOption, SequenceNumberConditionSupport,
-    SequenceNumberOption, SequenceNumberSupport, SnapshotOption, SnapshotRequired, SnapshotSupport,
+    MaxResultsOption, MaxResultsSupport, Metadata, MetadataOption, MetadataSupport,
+    NextMarkerOption, NextMarkerSupport, PageBlobLengthRequired, PageBlobLengthSupport,
+    PrefixOption, PrefixSupport, ProposedLeaseIdOption, ProposedLeaseIdRequired,
+    ProposedLeaseIdSupport, RangeOption, RangeRequired, RangeSupport,
+    SequenceNumberConditionOption, SequenceNumberConditionSupport, SequenceNumberOption,
+    SequenceNumberSupport, SnapshotOption, SnapshotRequired, SnapshotSupport,
     SourceContentMD5Option, SourceContentMD5Support, SourceLeaseIdOption, SourceLeaseIdSupport,
-    SourceUrlRequired, SourceUrlSupport, StoredAccessPolicy, StoredAccessPolicyList, TimeoutOption,
-    TimeoutSupport, UserAgent, UserAgentSupport, EMPTY_BODY,
+    SourceUrlRequired, SourceUrlSupport, StoredAccessPolicy, StoredAccessPolicyList, Timeout,
+    TimeoutOption, TimeoutSupport, UserAgent, UserAgentSupport, EMPTY_BODY,
```

Comment:
Can we delete `TimeoutOption` and any other traits that we're no longer using? Eventually most of this file can be deleted.

Reply:
Yes as soon as they have been removed from every request builder we'll removed them. The other PR (#119) will address the rest of storage request builders (I will have to merge this one first because it contains the `Timeout` struct).